### PR TITLE
Try to break path out into an object

### PR DESCRIPTION
**Pull request by aellispierce**

### CODE REVIEW

File: app/application.js [hunk 1]
```diff
@@ -1,26 +1,33 @@
 "use strict";
 
-var root = 'http://localhost:4000/api';
-var origin = window.location.origin;
+const root = 'http://localhost:4000/api';
+const origin = window.location.origin;
+
+function link(id, text){
+  return `<a href= ${origin}/announcement.html?id=${id}>${text}<br/><br/></a>`
+}
```
Comment (aellispierce) [inline app/application.js]: the only way i could get it to know what `link` was was by taking it out of the var App

File: app/application.js [hunk 2]
```diff
@@ -1,26 +1,33 @@
...
+    announcements() {
+      `${root}/announcements`
+    }
 
+  },
+  constableApi: class ConstableApi {
     getAnnouncements(){
       $.ajax({
-        url: root + '/announcements',
+        url: Path.announcements(),
```
Comment (aellispierce) [inline app/application.js]: this doesn't work. i've also tried `path.announcements()` and `this.path.announcements()`, to no avail

File: app/application.js [hunk 3]
```diff
@@ -1,26 +1,33 @@
...
+  return `<a href= ${origin}/announcement.html?id=${id}>${text}<br/><br/></a>`
+}
 
 var App = {
   init: function init() {
     $("body").prepend(
         `<a href = ${origin}/new.html> + </a>`)
   },
-  constableApi: class ConstableApi {
+  path: class Path {
```
Comment (drapergeek) [inline app/application.js]: I'm not sure about how to best structure these but I don't think this should be part of `App` anymore. It's weird to ahve a property that returns a whole class here.

File: app/application.js (hunk 1 above)
Comment (drapergeek) [inline app/application.js]: I think pulling it out of app is fine but I would make this into a function on the `Path` class if possible.

**Comment by aellispierce** [comment]
@drapergeek this makes a lot more sense to me now not having the classes as properties on App! Let me know your thoughts

**Comment by drapergeek** [comment]
This is a nice first pass!

There are some cleanups but I think this should bemerged and come back to them.

**Pull request merged**